online.py

```diff
@@ -19,14 +45,15 @@ def get_all_offline():
     return offline
 
 for user in database.get_all_offline():
-    perm = 'dev' if user['SPermission'][0] == 'Debugpass' else 'mod' if\
-        user['SPermission'][0] == 'adminpass' else 'mod'\
-            if user['SPermission'][0] == 'modpass' else None
+    # print(user)
+    perm = 'dev' if user['perm'][0] == 'Debugpass' else 'admin' if\
+        user['perm'][0] == 'adminpass' else 'mod'\
+            if user['perm'][0] == 'modpass' else None
     
     status = 'offline' if user['status'] != 'offlne-locked' else 'offline-locked'
             
     users_list[user['userid']] = {'username': user['displayName'],
-                                  'status': status, 'perm': perm}
+                                  'status': status, 'perm': perm, 'unread': {}}
```

Comment:
I was unable to get this to grab the unread data from the data base and link it to the users on start up. This system is quite confusing, still is for me, if you would like to tackle this one let me know and ill explain the code in detail for you.

Reply:
Resolve when read @cserver45